desktop/TransactionManager.ts

```diff
@@ -140,15 +144,14 @@ class TransactionManager {
       )
       .then((txs) => txs.map(this.handleNewTx(publicKey)))
       .catch((err) => {
-        console.log('grpc TransactionState', err); // eslint-disable-line no-console
         this.logger.error('grpc TransactionState', err);
       });
   });
 
   private subscribeAccount = (account: AccountWithBalance): void => {
     const { publicKey } = account;
     // Cancel account Txs subscription
-    this.txStateStream[publicKey] && this.txStateStream[publicKey]();
+    this.txStateStream[publicKey] && this.txStateStream[publicKey]?.();
```

Comment:
```suggestion
    this.txStateStream[publicKey] && this.txStateStream[publicKey]?.();
```
```suggestion
    this.txStateStream[publicKey]?.();
```
<img width="822" alt="Screenshot 2022-06-24 at 16 49 18" src="https://user-images.githubusercontent.com/54288612/175571424-da9a6116-9e02-4567-b2d3-
9e39b2d8cca7.png">